src/containers/UsersList.js

```diff
@@ -68,6 +68,16 @@ export class UsersList extends Component {
     })
   };
 
+  handleSearchInput = e => {
+    const users = this.props.users.filter(user => {
+      const name = `${user.first_name} ${user.last_name}`
+
+      return name.includes(e.target.value)
```

Comment:
What do you think of sanitizing the user input (trimming and lowercasing it) and lowercasing the user name?

Reply:
Will do, thanks.